custom_components/tuya_local/devices/smart_powerstrip_3_outlet_energy.yaml

```diff
@@ -0,0 +1,154 @@
+name: 3 Outlet Smart Powerstrip + Energy monitor
+products:
+  - id: y3vaekzobpkbcj0j
+    name: Eightree Powerstrip
+primary_entity:
+  entity: switch
+  class: outlet
+  name: Outlet 1
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 41
+      type: string
+      name: cycle_timer
+      optional: true
+    - id: 42
+      type: string
+      name: random_timer
+      optional: true
+secondary_entities:
+  - entity: switch
+    class: outlet
+    name: Outlet 2
+    dps:
+      - id: 2
+        type: boolean
+        name: switch
+  - entity: switch
+    class: outlet
+    name: Outlet 3
+    dps:
+      - id: 3
+        type: boolean
+        name: switch
+  - entity: number
+    category: config
+    name: Timer 1
+    icon: "mdi:timer"
+    dps:
+      - id: 9
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    category: config
+    name: Timer 2
+    icon: "mdi:timer"
+    dps:
+      - id: 10
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: number
+    category: config
+    name: Timer 3
+    icon: "mdi:timer"
+    dps:
+      - id: 11
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 60
+            step: 60
+  - entity: sensor
+    category: diagnostic
+    name: Energy
+    dps:
+      - id: 17
+        name: sensor
+        type: integer
+        unit: Wh
+        optional: true
+      - id: 25
+        type: integer
+        name: calibration
+        optional: true
+  - entity: sensor
+    category: diagnostic
+    class: current
+    dps:
+      - id: 18
+        name: sensor
+        type: integer
+        class: measurement
+        force: true
```

Comment:
Only use `force: true` if it is necessary to get the device to report sensor data, as some devices that don't need it have problems with the forced polling.  Generally if `force: true` is needed, `optional: true` will be too, because if the device is not reporting those values all the time, then they need to be ignored for detection.

Reply:
Double checked the use of `force: true` with the device, it still works as intended without the use of `force: true`